src/domain/alerts/alerts.domain.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Global, Module } from '@nestjs/common';
+import { AlertsApiModule } from '@/datasources/alerts-api/alerts-api.module';
+import { AlertsRepository } from '@/domain/alerts/alerts.repository';
+import { IAlertsRepository } from '@/domain/alerts/alerts.repository.interface';
+import { DelayModifierDecoder } from '@/domain/alerts/contracts/delay-modifier-decoder.helper';
+import { SafeDecoder } from '@/domain/alerts/contracts/safe-decoder.helper';
+import { MultiSendDecoder } from '@/domain/alerts/contracts/multi-send-decoder.helper';
+
+@Global()
```

Comment:
Why is this module a global module?

Reply:
No need, I've changed it in [f51fd4b](https://github.com/safe-global/safe-client-gateway/pull/879/commits/f51fd4b3fcf4eb16814563c84d2e497c239a76d0). Thanks!